internal/application/filesystem.go

```diff
@@ -518,9 +521,20 @@ func runTranslate(appName string, artifactpath string, translatech chan string,
 		close(outch)
 		m2kqaservermetadatapath := filepath.Join(artifactpath, m2kQAServerMetadataFile)
 		os.RemoveAll(m2kqaservermetadatapath)
+		srcDirectoryPath := filepath.Join(appName, assetsDirectory, srcDirectory)
+		artifactSrcDirectoryPath := filepath.Join(appName, artifactsDirectoryName, artifactName, appName, containersDirectory)
+		filename := filepath.Join(appName, artifactsDirectoryName, artifactName, appName, additionalFilename)
```

Comment:
We can possibly compute filepath.Join(appName, artifactsDirectoryName, artifactName, appName) path, and pass it to the function, and the function can then compute the two paths.

Reply:
Yes sure, I will update the code.

Reply:
I have made the suggested changes. Please review it.